### DISCUSSION TITLE
How to select Event Type 5  to be marked on the BT curve ?

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1499] [General]
![cba6ff5800d4232e9b61eba24c96d6b](https://github.com/artisan-roaster-scope/artisan/assets/149906183/0fab7e9c-c57b-4f29-ae12-d7029299df91)

**Comment by Terracotta-6** [comment]
[24-03-09_1616.alog.txt](https://github.com/artisan-roaster-scope/artisan/files/14549762/24-03-09_1616.alog.txt)

**Comment by MAKOMO** [comment]
<img width="480" alt="Screenshot 2024-03-10 at 12 03 18" src="https://github.com/artisan-roaster-scope/artisan/assets/13909011/2af8c766-cda3-4acb-a0e2-35432a3d34dc">
<img width="439" alt="Screenshot 2024-03-10 at 12 03 24" src="https://github.com/artisan-roaster-scope/artisan/assets/13909011/b25b7dd9-8577-400b-93fb-1e355e3bd79e">

**Reply by Terracotta-6** [reply]
Thank you for your answer.

**Comment by beanoccio** [comment]
I can't find any documentation for Event 5...
![image](https://github.com/user-attachments/assets/240b7a7a-8856-48a5-8a30-faec374d6f91)

**Reply by MAKOMO** [reply]
There are 4 "typed" events and one "untyped" event. See under [Events](https://artisan-scope.org/docs/events/).

**Comment by beanoccio** [comment]
Yes, I know this documentation about events, but I can't find anything in it about this "untyped" Event 5.

**Comment by roasterdave** [comment]
Untyped events are not shown as a curve.  These events appear in the graph as flags on the temperature curve when the "5" checkbox is ticked on the Config>> Events>>Config-tab.  They always appear in the events table (Roast Properties>> Events-tab).   There can be any number of these events but each is independent of the others.  

These events can be added during recording with the "Event" button, which itself is enabled by the "Event Button" checkbox on Config>> Events>>Config-tab.  

They can be added and edited in the graph view using the Events Editor, which is shown or hidden using the menu View>> Events Editor.  They can also be added and edited in the table on the Roast Properties>> Events-tab.  

Untyped events are useful for recording ad hoc information like outlier pops prior to first crack, or trier pulls, or anything else noteworthy.

On the Config>> Events>> Config-tab, ticking "Show on BT" will attach the untyped events flags to the BT curve.  Unticked and they will attache to the higher of ET or BT.  Control the width of the flag by ticking "Desc." and adjusting the the number of characters to display. 

![image](https://github.com/user-attachments/assets/39ea4a0f-fa43-42d9-b293-a80bdcad1478)

**Comment by beanoccio** [comment]
Ok, now it's clear. I always had this checkbox set to True and didn't miss anything.
Thank you very much for the detailed explanation.
